frontend/public/components/import-yaml.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { connectToFlags, FLAGS } from '../features';
+import { PodModel } from '../models';
+import { CreateYAML } from './create-yaml';
+
+class ImportYamlPage_ extends React.PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const match = {url: '', params: {ns: 'default', plural: 'pods'}, isExact: true, path: ''};
+    return <React.Fragment>
+      <div className="yaml-editor-header">
+        <div>Import YAML</div>
+        <div className="yaml-editor-subheader">Create or replace resources from their YAML or JSON definitions. Use the template below or browse to upload a file.</div>
+      </div>
+      <CreateYAML match={match} kindsInFlight={false} kindObj={PodModel} showHeader={false} />
```

Comment:
`PodModel` doesn't seem right... It would be better to make that an optional prop and not specify it for this case.

Reply:
If it is not specified, then there would be nothing in the editor, when it loads.
It doesn't seem like that was a state in any of the designs I was seeing. Does an empty editor give enough cues to the user?

Reply:
Presumably here you already have the YAML you want to use and don't need to work from a template. Having content prefilled might even get in the way since you have to select or delete it to paste something new.

We had some designs for drag n drop support where there's an empty state message saying to drag or paste a file. We don't necessarily need that for the first pass, though.